index.js.flow

```diff
@@ -57,11 +57,15 @@ type AssertContext = {
 	notDeepEqual<U>(value: U, expected: U, message?: string): void;
 	// Assert that function throws an error or promise rejects.
 	// @param error Can be a constructor, regex, error message or validation function.
-	throws(value: PromiseLike<mixed>, error?: ErrorValidator, message?: string): Promise<mixed>;
-	throws(value: () => void, error?: ErrorValidator, message?: string): mixed;
+	throws: {
+		(value: PromiseLike<mixed>, error?: ErrorValidator, message?: string): Promise<void>;
+		(value: () => mixed, error?: ErrorValidator, message?: string): void;
+	};
 	// Assert that function doesn't throw an error or promise resolves.
-	notThrows<U>(value: PromiseLike<U>, message?: string): Promise<U>;
-	notThrows(value: () => void, message?: string): void;
+	notThrows: {
+		notThrows<U>(value: PromiseLike<U>, message?: string): Promise<U>;
```

Comment:
Interesting. This isn't actually supposed to return the value, but looking at the code I think it does…